yt_dlp/extractor/gettr.py

```diff
@@ -108,3 +109,70 @@ def _real_extract(self, url):
             'duration': float_or_none(post_data.get('vid_dur')),
             'tags': post_data.get('htgs'),
         }
+
+
+class GettrStreamingIE(GettrIE):
+    IE_NAME = 'Gettr:streaming'
+    _VALID_URL = r'https?://(www\.)?gettr\.com/streaming/(?P<id>[a-z0-9]+)'
+
+    _TESTS = [{
+        'url': 'https://gettr.com/streaming/psoiulc122',
+        'info_dict': {
+            'id': 'psoiulc122',
+            'ext': 'mp4',
+            'description': 'md5:56bca4b8f48f1743d9fd03d49c723017',
+            'view_count': int,
+            'uploader': 'Corona Investigative Committee',
+            'uploader_id': 'coronacommittee',
+            'duration': 5180184,
+            'thumbnail': r're:^https?://.+',
+            'title': 'Day 1: Opening Session of the Grand Jury Proceeding',
+            'timestamp': 1644080997164,
+        }
+    }, {
+        'url': 'https://gettr.com/streaming/psfmeefcc1',
+        'info_dict': {
+            'id': 'psfmeefcc1',
+            'ext': 'mp4',
+            'title': 'Session 90: "The Virus Of Power"',
+            'view_count': int,
+            'uploader_id': 'coronacommittee',
+            'description': 'md5:98986acdf656aa836bf36f9c9704c65b',
+            'uploader': 'Corona Investigative Committee',
+            'thumbnail': r're:^https?://.+',
+            'duration': 21872507,
+            'timestamp': 1643976662858,
+        }
+    }]
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        video_info = self._download_json('https://api.gettr.com/u/live/join/%s' % video_id, video_id, data={})['result']
+
+        live_info = video_info['broadcast']
+        live_url = url_or_none(live_info.get('url'))
+
+        formats, subtitles = self._extract_m3u8_formats_and_subtitles(
+            live_url, video_id, ext='mp4',
+            entry_protocol='m3u8_native', m3u8_id='hls', fatal=False) if live_url else []
+
+        thumbnails = []
+        for thumbnail in try_get(video_info, lambda x: x['postData']['imgs']) or []:
+            thumbnails.append({
+                'url': urljoin(self._MEDIA_BASE_URL, thumbnail),
+            })
+
+        return {
+            'id': video_id,
+            'title': try_get(video_info, lambda x: x['postData']['ttl']),
+            'description': try_get(video_info, lambda x: x['postData']['dsc']),
+            'formats': formats,
+            'subtitles': subtitles,
+            'thumbnails': thumbnails,
+            'uploader': try_get(video_info, lambda x: x['liveHostInfo']['nickname']),
+            'uploader_id': try_get(video_info, lambda x: x['liveHostInfo']['_id']),
+            'view_count': int_or_none(live_info.get('viewsCount')),
+            'timestamp': int_or_none(live_info.get('startAt')),
+            'duration': int_or_none(live_info.get('duration')),
```

Comment:
```suggestion
            'duration': float_or_none(live_info.get('duration'), scale=1000),
```
Duration is given in miliseconds, while info_dict should set `duration` to seconds (